zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/X509AuthenticationConfig.java

```diff
@@ -367,6 +391,25 @@ private Set<String> loadOpenReadAccessPathPrefixes() {
         .filter(str -> str.length() > 0).collect(Collectors.toSet());
   }
 
+  /**
+   * Helper method for Znode Group Acl feature
+   * Get open read access path prefixes from config
+   * @return A set of path prefixes
```

Comment:
copy/paste error? not related to "Get open read access path prefixes from config"